assets/stylesheets/sass/main/main.scss

```diff
@@ -128,6 +133,32 @@ figure {
     }
   }
 
+  .built-by{
+    color: $light-gray;
+    font-size: 0.9em;
+    margin-top: 5em;
+
+    p {
+      text-align:center;
+    }
+
+    a {
+      text-decoration:none;
+      color:#b3b3b3;
+
+      &:hover{
```

Comment:
Selector should have depth of applicability no greater than 2, but was 3<br>Opening curly brace `{` should be preceded by one space